erpnext/stock/doctype/pick_list/pick_list.json

```
@@ -114,7 +114,7 @@
  ],
  "is_submittable": 1,
  "links": [],
- "modified": "2020-03-17 11:38:41.932875",
+ "modified": "2021-05-25 16:51:37.210932",
```

```suggestion
 "modified": "2020-03-17 11:38:41.932875",
```

schema not modified. So this change can be reverted.